iac-extensions/terraform/src/test/java/org/sonar/iac/terraform/plugin/TerraformProvidersTest.java

```diff
@@ -119,7 +119,7 @@ void assess_without_provider_version() {
 
   @Test
   void check_property_registration() {
-    long registeredProperties = TerraformSettings.getProperties().stream()
+    long registeredProperties = TerraformSettings.getGeneralProperties().stream()
```

Comment:
With change in `TerraformExtension` this would have to be updated as well i think

Reply:
Not this one, but I added a 2 tests to cover my changes